src/pudl/transform/eia861.py

```diff
@@ -2223,11 +2240,15 @@ def reliability(tfr_dfs):
     transformed_r = (
         tidy_r.assign(
             outages_recorded_automatically=lambda x: (
-                _make_yn_bool(x.outages_recorded_automatically.str.upper())),
+                _make_yn_bool(x.outages_recorded_automatically)
```

Comment:
Why did you remove `.upper()` here?

Reply:
Because I included both lower and upper case Y and N in the map, so that one doesn't have to remember to deal with the case here.